services/scripts/testy-snapshot.sh

```diff
@@ -41,9 +40,16 @@ main() {
         exit 1
     fi
 
+    # Retrieve the tags from the instance.
+    local _tags
+    _tags=$(aws ec2 describe-instances --instance-ids "$_instance_id" \
+        --query "Reservations[*].Instances[*].Tags[][]" --output json)
+    _tags="${_tags//\"/}"
```

Comment:
Can we add a comment why this is necessary?

Reply:
Yes, added in [a15ad4f](https://github.com/wiredtiger/testy/pull/50/commits/a15ad4f62de8b96fa9dd2295f6d533ff4b326486). Let me know if it's clear!